components/popover/index.js

```diff
@@ -109,10 +108,9 @@ class Popover extends Component {
 		// Related https://stackoverflow.com/questions/35522220/react-ref-with-focus-doesnt-work-without-settimeout-my-example
 		const focusNode = ( domNode ) => setTimeout( () => domNode.focus() );
 
-		// Find first tabbable node within content and shift focus, falling
-		// back to the popover panel itself.
-		const firstTabbable = focus.tabbable.find( this.contentNode.current )[ 0 ];
-		focusNode( firstTabbable ? firstTabbable : this.contentNode.current );
+		// Focus the popover panel so further tips are easily navigated to via the
```

Comment:
We should probably avoid mentioning tips because it's not specific to tips.

Reply:
Oh, yes, good call. I was too narrow in my documentation because it's where I tested it.